packages/frontend-2/components/viewer/gendo/Dialog.vue

```diff
@@ -9,39 +9,101 @@
           :alt="renderPrompt"
-          class="relative z-10 w-full h-full max-h-[70vh] max-w-[80vw] object-contain"
+          class="relative z-10 w-full h-full max-h-[70vh] max-w-[80vw] object-contain rounded-xl"
         />
       </div>
-      <div class="relative z-10 flex gap-2">
-        <FormButton
-          :to="renderUrl"
-          external
-          target="_blank"
-          download
-          color="outline"
-          :icon-left="ArrowDownTrayIcon"
-        >
-          Download
-        </FormButton>
-        <FormButton color="outline" :icon-left="XMarkIcon" @click="isOpen = false">
-          Close
+      <div
+        v-if="!hasFeedback"
+        class="bg-foundation rounded-md p-2 relative z-10 flex items-center justify-between gap-2 mb-4 w-full"
+      >
+        <div class="flex gap-2 h-8">
+          <FormButton :to="renderUrl" external target="_blank" download color="outline">
+            Download
+          </FormButton>
+          <FormButton color="subtle" @click="isOpen = false">Close</FormButton>
+        </div>
+        <div class="flex items-center gap-2">
+          <p class="text-body-xs text-foreground-2">
+            What do you think about this render?
+          </p>
+          <FormButton
+            :icon-left="HandThumbUpIcon"
+            hide-text
+            color="subtle"
+            @click="handleThumbsUp"
+          >
+            Thumbs up
+          </FormButton>
+          <FormButton
+            :icon-left="HandThumbDownIcon"
+            hide-text
+            color="subtle"
+            @click="handleThumbsDown"
+          >
+            Thumbs down
+          </FormButton>
+        </div>
+      </div>
+      <div v-else class="flex items-center gap-2">
+        <p class="text-body-xs text-foreground-2">Thanks for your feedback!</p>
+        <FormButton color="outline" @click="isFeedbackDialogOpen = true">
+          Give detailed feedback
         </FormButton>
       </div>
     </div>
+    <FeedbackDialog
+      v-model:open="isFeedbackDialogOpen"
+      type="gendo"
+      intro="Help us improve Gendo AI renders. Share your thoughts about the quality, prompts that work well, or features you'd like to see."
+      :metadata="{
+        initialFeedback: initialFeedback,
+        render_url: renderUrl,
+        prompt: renderPrompt
+      }"
+    />
   </LayoutDialog>
 </template>
 
 <script setup lang="ts">
-import { XMarkIcon, ArrowDownTrayIcon } from '@heroicons/vue/24/solid'
+import { HandThumbUpIcon, HandThumbDownIcon } from '@heroicons/vue/24/solid'
+import { useMixpanel } from '~/lib/core/composables/mp'
+
+const mixpanel = useMixpanel()
 
-defineProps<{
+const props = defineProps<{
   renderUrl?: string
   renderPrompt?: string
 }>()
 
 const isOpen = defineModel<boolean>('open', { required: true })
+
+const isFeedbackDialogOpen = ref(false)
+const hasFeedback = ref(false)
+const initialFeedback = ref<'positive' | 'negative' | null>(null)
+
+const handleThumbsUp = () => {
```

Comment:
You could also have one function for both (and make render_url snakeCase, unless we already use it, and end the the event with a past-tense verb)

```
type FeedbackValue = 'positive' | 'negative';

const initialFeedback = ref<FeedbackValue>()

const handleFeedback = (value: FeedbackValue) => {
  initialFeedback.value = value
  mixpanel.track('Gendo Render Feedback Given', {
    feedback: value,
    prompt: props.renderPrompt,
    renderUrl: props.renderUrl
  })
  
  hasFeedback.value = true
}
```